app/presenters/account_show_presenter.rb

```diff
@@ -34,10 +36,44 @@ def show_gpo_partial?
     user.gpo_verification_pending_profile?
   end
 
+  def show_ipp_partial?
+    user.pending_in_person_enrollment.present?
+  end
+
   def showing_any_partials?
+    show_password_reset_partial? ||
+      show_gpo_partial? || show_ipp_partial? || show_pii_partial?
+  end
+
+  def showing_alerts?
     show_service_provider_continue_partial? ||
-      show_password_reset_partial? ||
-      show_gpo_partial?
+      show_password_reset_partial?
+  end
+
+  def show_unverified?
+    show_password_reset_partial? &&
+      (!show_ipp_partial? || !show_gpo_partial?)
+  end
+
+  def find_sp_issuer
+    sp_name = 'APP_NAME'
```

Comment:
Should this be the literal `APP_NAME` constant instead of `'APP_NAME'` as a string?

```suggestion
    sp_name = APP_NAME
```

Reply:
> Should this be the `APP_NAME` constant instead of `'APP_NAME'` as a string?

This was removed in ac9737e087.